src/actions/account.js

```diff
@@ -192,32 +191,31 @@ export const {
     checkNewAccount,
     createNewAccount,
     checkAccountAvailable,
-    clear,
     clearCode
 } = createActions({
     INITIALIZE_RECOVERY_METHOD: [
         wallet.initializeRecoveryMethod.bind(wallet),
-        () => defaultCodesFor('account.initializeRecoveryMethod')
+        () => ({})
```

Comment:
@marcinbodnar  is empty function needed here for some reason or we can just omit `meta` for such cases?

Reply:
also what is the implied meaning – does this mean that we expect whoever dispatched this action to handle errors locally?

Reply:
>is empty function needed here for some reason or we can just omit meta for such cases?

We can omit it, but let me take care of that while refactoring redux state

> also what is the implied meaning – does this mean that we expect whoever dispatched this action to handle errors locally without error alert?

Information about action success or failure will be stored anyway automatically in `actionStatus` object - by adding or not adding `showAlert` we decide do we want to show an alert about this error/success or not.